README.md

```diff
@@ -1,2 +1,67 @@
 # consert-website
-A website consert
+
+
+ This is a Microverse consert website project.
+ 
+## Built With
+
+- HTML
+- css
+- javascript
+## Live Demo (not available)
```

Comment:
- [ ] There should be a  video of a maximum length of  5 minutes. describing the features of your project, Kindly add that